package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-azureappservice",
```

Comment:
Not sure if you were just waiting for it to be published, but make sure to update the version of the appservice npm package in this file